biojava-structure/src/main/java/org/biojava/nbio/structure/Calc.java

```diff
@@ -763,7 +764,11 @@ public static final void shift(Group group, Atom a) {
 	}
 
 	/**
+<<<<<<< HEAD
+	 * Returns the centroid of the set of atoms.
+=======
```

Comment:
Fixed! My Effective Java edition only has 11 chapters (I had to check to realize it was a joke).